test/addons/openssl-key-engine/test.js

```diff
@@ -46,16 +46,10 @@ const server = https.createServer(serverOptions, common.mustCall((req, res) => {
   };
 
   const req = https.request(clientOptions, common.mustCall((response) => {
-    let body = '';
-    response.setEncoding('utf8');
-    response.on('data', (chunk) => {
-      body += chunk;
-    });
-
-    response.on('end', common.mustCall(() => {
+    common.collectStream(response).then((body) => {
       assert.strictEqual(body, 'hello world');
       server.close();
-    }));
+    });
```

Comment:
Likewise... for all of these tests, the then handler needs a `common.mustCall()`.